manga-loader.user.js

```diff
@@ -1571,6 +1616,10 @@ var getViewer = function(prevChapter, nextChapter) {
         'z-index': '100',
         'position': 'relative'
       }, '.ml-counter'),
+      navdivCss = toStyleStr({
+        'position': 'sticky',
```

Comment:
Not sure how I feel about the sticky nav, it overlaps the image and kind of distracts a bit.